numpy/_core/src/umath/loops_comparison.dispatch.cpp

```diff
@@ -0,0 +1,638 @@
+#include "numpy/npy_common.h"
+#include "common.hpp"
+#include "simd/simd.h"
+#include "loops_utils.h"
+#include "loops.h"
+#include <hwy/highway.h>
```

Comment:
```suggestion
#include "simd/simd.hpp"
```
will be used instead, note that Highway SIMD is only available when optimization is enabled (default through meson flag)